### PR TITLE
Bar Map Tweaks and Audioconsole Variant

### PR DESCRIPTION
**Pull request by SleepyGemmy**
this PR adds a wall-mounted variant of the audioconsole. it also adds proper bar stools and does some other bar area map tweaks.

ps: the icondiff bot is a bit bugged so it shows a stray white pixel in the top-left corner.

### CODE REVIEW

**Comment by SleepyGemmy** [comment]
!review

**Comment by alsoandanswer** [comment]
I think the small audioconsole and the wall console could do without the curve. It looks kind of scuffed on the walls.

**Review by alsoandanswer — APPROVED** [review]

File: code/game/machinery/jukebox.dm [hunk 1]
```diff
@@ -249,7 +249,7 @@ datum/track/New(var/title_name, var/audio)
 		new/datum/track("Don't Rush", 'sound/music/audioconsole/DontRush.ogg'),
 		new/datum/track("Phoron Will Make Us Rich", 'sound/music/audioconsole/PhoronWillMakeUsRich.ogg'),
 		new/datum/track("Amsterdam", 'sound/music/audioconsole/Amsterdam.ogg'),
-		new/datum/track("when", 'sound/music/audioconsole/When.ogg'),
+		new/datum/track("Whenever", 'sound/music/audioconsole/When.ogg'),
```
Comment (alsoandanswer) [inline code/game/machinery/jukebox.dm:252]:
```suggestion
		new/datum/track("when", 'sound/music/audioconsole/When.ogg'),
```

this is the actual name of the song don't change it thanks

Comment (SleepyGemmy) [reply code/game/machinery/jukebox.dm]: addressed.

**Review by JohnWildkins — APPROVED** [review]

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]
Not a fan of the way you changed the flooring in the western bar section. It's supposed to look roomy/comfortable and not just be an extension of the normal bar's color/floor scheme. The random railings and one by one spaces will also get in the way a lot.

**Pull request merged**